app/code/core/Mage/AdminNotification/Helper/Data.php

```diff
@@ -96,38 +80,10 @@ public function getUnreadNoticeCount($severity)
      *
      * @param bool $withExt
      * @return string
+     * @deprecated v19.4.16
      */
     public function getPopupObjectUrl($withExt = false)
     {
-        if (is_null($this->_popupUrl)) {
-            $this->_popupUrl = 'https://' . Mage::getStoreConfig(self::XML_PATH_POPUP_URL);
-        }
-        return $this->_popupUrl . ($withExt ? '.swf' : '');
-    }
-
-    /**
-     * Check is readable Popup Notification Object
-     * @deprecated after 1.4.2.0
-     *
-     * @return bool
-     */
-    public function isReadablePopupObject()
-    {
-        if (is_null($this->_popupReadable)) {
-            $this->_popupReadable = false;
-            $curl = new Varien_Http_Adapter_Curl();
-            $curl->setConfig(array(
-                'timeout'   => 2
-            ));
-            $curl->write(Zend_Http_Client::GET, $this->getPopupObjectUrl(true));
-            if ($curl->read()) {
-                if ($curl->getInfo(CURLINFO_HTTP_CODE) == 200) {
-                    $this->_popupReadable = true;
-                }
-            }
-
-            $curl->close();
-        }
-        return $this->_popupReadable;
+        return '';
```

Comment:
`getPopupObjectUrl return ''` and `isReadablePopupObject return false`, no?

Reply:
I'm not sure I understood the question correctly. The getPopupObjectUrl method was returning a URL so I kept it returning a string (empty one), while the isReadablePopupObject method is now completely removed.

Reply:
oh, ok, no problem!